frontend/app/views/fragments/page/elevatedCloserGuardianBody.scala.html

```diff
@@ -0,0 +1,51 @@
+@import model.OrientatedImages
+@import views.support.Asset
+@import configuration.Videos
+
+<div class="l-constrained elevated-closer-guardian-body">
+
+    <div class="elevated-closer-guardian-body__title u-content-width--left">
+        <h1 class="elevated-closer-guardian-body__title__h1">Supporters get closer</h1>
```

Comment:
Does this text need to be separated into 2 elements? It doesn't make sense semantically. I'd also suggest not coupling class names to the tag type.